src/app/rns-sdk.js

```diff
@@ -50,4 +49,4 @@ export const rns = (
 
 export const resolver = (
   signer = defaultSigner,
-) => new AddrResolver(resolverAddress, signer);
+) => new AddrResolver(registryAddress, signer);
```

Comment:
just add a comment behind the reasoning behind this lol

Reply:
Okay, sure!